docs/src/main/asciidoc/sns.adoc

```diff
@@ -159,3 +159,45 @@ public void receiveQueueMessages(Person person) {
 
 When using the logical names like in the example above, the stack can be created on different environments without any
 configuration or code changes inside the application.
+
+
+=== IAM Permissions
+Following IAM permissions are required by Spring Cloud AWS:
+
+[cols="2"]
+|===
+| To publish notification to topic
+| `sns:Publish`
+
+| To publish notification you will also need
+| `SNS:ListTopics`
```

Comment:
There is an inconsistency between lowercase sns and upercase SNS

Reply:
FIxed